extensions-core/druid-ranger-security/pom.xml

```diff
@@ -193,7 +193,14 @@
             </exclusions>
             <scope>compile</scope>
         </dependency>
-
+        <!-- This is a transitive dependency of ranger-plugins-audit
+             added here to replace bloated bundle sdk, remove this if /when
+             apache ranger replaces bundle-skd with logs-sdk  -->
+        <dependency>
+            <groupId>com.amazonaws</groupId>
+            <artifactId>aws-java-sdk-logs</artifactId>
+            <version>1.12.669</version>
```

Comment:
nit, any reason we don't use ${aws.sdk.version} here and align the versions across the board?

Reply:
fixed